src/type/impls.rs

```diff
@@ -576,6 +576,104 @@ const _: () = {
         matrix2: DMat2,
         translation: DVec2,
     }
+
```

Comment:
Is it possible for you to merge in `main` and update these impls to follow the way they do them. Eg. `remote = glam::{type}`.

It would look like the way the glam remote impls work in `main` is just bugged.

Reply:
Now it works :) Thanks 

Would you mind if I add some `bevy_ecs` Type support (e.g. for `Entity`) in another MR? Or is that a to specific crate?

Reply:
Yeah go for it!